docs/installation/manage-viam-agent.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Manage viam-agent"
+linkTitle: "Manage viam-agent"
+weight: 80
+no_list: true
+type: docs
+draft: false
+images: ["/installation/thumbnails/manage.png"]
+imageAlt: "Manage viam-agent"
+description: "Control and troubleshoot viam-agent."
+---
+
+[`viam-agent`](/configure/agent/) is installed as a `systemd` service named `viam-agent`.
+
+- To start `viam-agent`:
+
+  ```sh {class="command-line" data-prompt="$"}
+  sudo systemctl start viam-agent
+  ```
+
+- To stop `viam-agent`:
+
+  {{< alert title="Alert" color="note" >}}
+  When you stop `viam-agent`, the agent will stop `viam-server` as well.
+  {{< /alert >}}
+
+  ```sh {class="command-line" data-prompt="$"}
+  sudo systemctl stop viam-agent
+  ```
+
+- To restart `viam-agent`:
+
+  {{< alert title="Alert" color="note" >}}
+  When you restart `viam-agent`, the agent will restart `viam-server` as well.
+  {{< /alert >}}
+
+  ```sh {class="command-line" data-prompt="$"}
+  sudo systemctl restart viam-agent
+  ```
+
```

Comment:
This should probably also include the enable and disable commands (similar to what's on the viam-server version of this page) The commands are almost the same, "viam-server" just becomes "viam-agent" in each.

Reply:
Still think this would be good to add, especially as this is now linked from the alert about "Automatic Startup"